.prettierrc

```diff
@@ -1,12 +1,4 @@
 {
-    "arrowParens": "always",
```

Comment:
Why are we making all these other changes to prettier?

Reply:
You can find out more if you look at the individual commits, as there's one commit for each logical change.

In short a lot of them are the defaults, except `printWidth` and `trailingComma` which are the changes by this PR

Reply:
Not removing the default ones would make this PR easier to review/less scary